pkg/router/template/router.go

```diff
@@ -553,6 +555,10 @@ func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
 		if hasRequiredEdgeCerts(cfg) {
 			return true
 		} else {
+			if cfg.TLSTermination == routeapi.TLSTerminationReencrypt && hasReencryptDestinationCACert(cfg) {
+				return true
+			}
+
 			msg := fmt.Sprintf("a %s terminated route with host %s does not have the required certificates.  The route will still be created but no certificates will be written",
```

Comment:
This message probably belongs above the new if statement since it is still applicable to a reencrypt route using the wild card certs.  It should probably be reworded though to make sure it is clearly talking about the edge certificates.